app/src/main/res/values/strings.xml

```diff
@@ -946,7 +946,10 @@
 
   <!-- Drug stock reminder notification -->
   <string name="drug_stock_reminder_notification">Complete your pending drug stock report for %1$s</string>
-  
+  <string name="drug_stock_reminder_card_title">Drug Stock</string>
+  <string name="drug_stock_reminder_card_content">Complete your pending drug\nstock report for %1$s</string>
```

Comment:
We don't need to set a line break (\n) here. Android will automatically wrap it as required. Design also doesn't indicate it as 2 different lines. So, we can remove it.